TestApp/Host/main.c

```diff
@@ -23,6 +23,32 @@
 #include "AWSteria_Platform.h"
 #include "AWSteria_Host_lib.h"
 
+void *AWSteria_Host_state = NULL;
+
+#ifdef USE_DRM
+#include "accelize/drmc.h"
+#include <stdio.h>
+
+// Define functions to read and write FPGA registers to use them as
+// callbacks in DrmManager.
+#define drm_controller_base_addr 0
+
+
+int read_register( uint32_t offset, uint32_t* value, void* user_p ) {
```

Comment:
Should we name this 'read_drm"register' (and similarly, below, for 'write_drm_register', since these are DRM module-specific? We'll be using the AXI4-Lite at other addresses for other (non-DRM-related) registers?